root/account/Donation.js

```diff
@@ -24,11 +24,16 @@ const Donation = ({days, nag}: Props) => (
       ? (
         <>
           <p>
-            {l('We have not received a donation from you recently. If you have just made a PayPal donation, then we have not received a notification from PayPal yet. Please wait a few minutes and reload this page to check again.')}
+            {l(`We have not received a donation from you recently. If you have
+                just made a PayPal donation, then we have not received a
+                notification from PayPal yet. Please wait a few minutes and
+                reload this page to check again.`)}
           </p>
           <p>
-            {l('If you would like to make a donation, {donate|you can do that here}. If you have donated, but you are still being nagged, please {contact|contact us}.',
-              {contact: CONTACT_URL, donate: DONATE_URL})}
+            {l(`If you would like to make a donation, {donate|you can do that
+                here}. If you have donated, but you are still being nagged,
+                please {contact|contact us}.`,
+            {contact: CONTACT_URL, donate: DONATE_URL})}
```

Comment:
I'm honestly not sure why eslint wants this object indented outside of the enclosing `{}` - that seems like a bug to me. For example, you wouldn't write:

```JSX
const x = (
  <div>
    {hey('whats',
    'up')}
  </div>
);
```

You'd indent `'up'` in like you would in any other case. In fact, eslint complains about that in this example...it's only buggy if you make `'whats'` a multiline string.

Reply:
It's always asked for this in multiline strings, yeah. I assumed there was a reason, but we could always change it - should we report upstream and ask first?

Reply:
I figured out how to get it working with https://github.com/metabrainz/musicbrainz-server/pull/953

With that it'll want all arguments aligned with each other, as in the opening `` ` `` and opening `{` here in each argument:

```JSX
{l(`multiple
    lines are {cool}`,
   {cool: true})}
```

Reply:
Neat! Let's get that one merged, then I can use it to fix this and a bunch of other cases like this one too.

Reply:
Updated these